solutions/day-27/src/components/ColorCell.js

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import "../styles/ColorCell.css";
+
+const ColorCell = ({ hexColor }) => {
+  const [copyClassName, setCopyClassName] = useState("copy");
+
+  const onCopy = () => {
+    navigator.clipboard.writeText(hexColor);
+    setCopyClassName("copied");
+    setTimeout(() => {
+      setCopyClassName("copy");
+    }, 2000);
+  };
+
+  return (
+    <div className="color-cell" style={{ backgroundColor: hexColor }}>
+      <small>{hexColor}</small>
+      <div className={`wrapper_copy ${copyClassName}`}>
+        <i className="fa fa-copy" onClick={onCopy}></i>
```

Comment:
What we usually do when an element has a behavior or is clickable, we wrap it with a button element. In that way, the a11y tools will read this as a button. I know you have not undergone any a11y training,  but please note that we must always consider a11y when building a component.

Reply:
Updated. Thanks. @ianabalus.